### ISSUE TITLE
Keeweb fails to install on Debian 11 (bullseye): unmet dependency libappindicator1

### ISSUE THREAD

**Issue by Zagur** [keeweb/keeweb#1876]
**Describe the bug**
Keeweb client depends on the libappindicator1 package that [has been deprecated](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=895037) and is not available in Debian 11 (bullseye). So, it is not possible to install the current original Keeweb deb.

**To Reproduce**
1. dpkg -i keeweb*.deb

**Environment**
Please copy all version information from Settings/Help

Debian 11

**Comment by antelle** [comment]
Hi! Does everything work if you run keeweb from a zip archive?

**Comment by Zagur** [comment]
> Hi! Does everything work if you run keeweb from a zip archive?

Yes, with the .zip file it works correctly. But the problem is when you try to install the .deb with dpkg.

**Comment by antelle** [comment]
Does the tray icon work without this dependency?

**Comment by carpntr** [comment]
I've run into this issue with a couple of apps (specifically discord and keeweb). Until this is resolved, here's a script for rebuilding debian packages with libayatana-appindicator. I've only tested this on debian 11.

https://gist.github.com/carpntr/2766531f204b957ab723ea3647d2b047

**Comment by VlaK0r** [comment]
> Hi! Does everything work if you run keeweb from a zip archive?

How to run keeweb from a zip archive? Tell me please.
